README.md

```diff
@@ -122,11 +135,112 @@ To install pgai from source on a PostgreSQL server:
 
    The `CASCADE` automatically installs `pgvector` and `plpython3u` extensions.
 
-### Use pgai
+### Usage of pgai
+
+The main features in pgai are:
  
-Now, use pgai to integrate AI from [Ollama](./docs/ollama.md), [OpenAI](./docs/openai.md), 
-[Anthropic](./docs/anthropic.md), and [Cohere](./docs/cohere.md). Learn how to [moderate](./docs/moderate.md) and
-[embed](./docs/delayed_embed.md) content directly in the database using triggers and background jobs.
+**Working with embeddings generated from your data:**
+* [Automatically create and sync vector embeddings for your data](#automatically-create-and-sync-llm-embeddings-for-your-data)
+* [Search your data using vector and semantic search](#search-your-data-using-vector-and-semantic-search)
+* [Implement Retrieval Augmented Generation inside a single SQL statement](#implement-retrieval-augmented-generation-inside-a-single-sql-statement) 
+
+**Leverage LLMs for data processing tasks:**
+You can use pgai to integrate AI from the following providers:
+- [Ollama](./docs/ollama.md)
+- [OpenAI](./docs/openai.md)
+- [Anthropic](./docs/anthropic.md)
+- [Cohere](./docs/cohere.md)
+- [Llama 3 (via Ollama)](/docs/ollama.md)
+
+Learn how to [moderate](/docs/moderate.md) content directly in the database using triggers and background jobs.
```

Comment:
This seems a bit backwards, I would put this section below the 3 use-cases (except moderation).
My idea would be:
1. Explain what you can build with pgai: automatic embeddings, Search, RAG, Moderation, (anything else?)
2. Explain the building blocks for these features: generate, embed, classify, tokenize, ...
    a. This needs a subsection of what is supported for which provider maybe

So you'd start from a higher abstraction and go into the details later.

**Automatic Embeddings** is a bit of an exception because it's a low level feature, but it might be useful for you even if you don't want to make use of the rest of pgai (e.g. you want to build a RAG in python instead of SQL).
The problem with that is, if you explain it only later an interested user would never find it since they e.g. already have a system for RAG or are set on using python so you would lose them at the SQL for semantic search/RAG.